src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.io.*;
+import java.util.*;
```

Comment:
Should list all imports explicitly, importing only classes that you need, i.e. import java.util.ArrayList;